### DISCUSSION TITLE
Make the File Library "Tags" Field Relational Instead of JSON

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#17220] [Feature Requests]
If you Google "what is the purpose of tags", you will find, for example, the following:

<img width="660" alt="image" src="https://user-images.githubusercontent.com/26592486/213244175-b66bd4a0-6f79-4cd6-8743-82ea05a3110a.png">

As it currently stands then, the "Tags" field in Directus is **useless**. Since it is implemented as a JSON field, I cannot search or filter by it (#13418, #12496). So honestly, in what scenario would I use it?

For this reason, I suggest that the "Tags" field be changed to a **relational implementation** until searching and filtering for JSON fields is implemented (#7277, #15889). I look forward to being able to filter on JSON fields, but in the meantime, I _really_ need to be able to filter through my image library somehow, because there is current no way to do so.

If not, I think the "Tags" field should be temporarily hidden, because it is extremely misleading.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
